gometalinter.json

```diff
@@ -4,6 +4,9 @@
   "Sort": ["linter", "severity", "path", "line"],
   "Exclude": [
       "cli/compose/schema/bindata.go",
+      "cli/command/stack/kubernetes/api/openapi",
+      "cli/command/stack/kubernetes/api/client",
+      ".*generated.*",
```

Comment:
These paths need to be added to the `ignore` section in  `codecov.yml` as well.

Reply:
Fixed